admin-console/src/main/webapp/src/components/EmailTemplates/EmailTemplateForm.tsx

```diff
@@ -18,53 +19,243 @@ const SkeletonIfLoading: React.FunctionComponent<SkeletonIfLoading> = props => (
-                    <CodeEditor
-                        isLineNumbersVisible
-                        code={ props.template?.data ?? '' }
-                        isMinimapVisible={ false }
-                        onChange={ data => props.updateTemplate({ data }) }
-                        height="300px"
-                    />
-                </SkeletonIfLoading>
-            </FormGroup>
-        </Form>
+        <>
+            <PageSection>
+                <Split hasGutter>
+                    <SplitItem isFilled>
+                        <Title headingLevel="h2">Result</Title>
+                    </SplitItem>
+                    <SplitItem>
+                        <Button onClick={ onRender }>Render</Button>
+                    </SplitItem>
+                </Split>
+                <RenderedTemplate { ...renderedProps }  />
+            </PageSection>
+            <Form>
+                <FormGroup
+                    label='Name'
+                    isRequired
+                    helperText='Enter a name for your template'
+                >
+                    <SkeletonIfLoading isLoading={ props.isLoading }>
+                        <TextInput
+                            type='text'
+                            id='name'
+                            name="name"
+                            value={ props.template?.name ?? '' }
+                            onChange={ name => props.updateTemplate({ name })}
+                        />
+                    </SkeletonIfLoading>
+                </FormGroup>
+                <FormGroup
+                    label='Description'
+                    isRequired
+                    helperText='Enter a brief description for your template'
+                >
+                    <SkeletonIfLoading isLoading={props.isLoading}>
+                        <TextInput
+                            type='text'
+                            id='description'
+                            name="description"
+                            value={ props.template?.description ?? '' }
+                            onChange={ description => props.updateTemplate({ description })}
+                        />
+                    </SkeletonIfLoading>
+                </FormGroup>
+                <FormGroup
+                    label="Subject"
+                    isRequired
+                >
+                    <SkeletonIfLoading isLoading={props.isLoading} height="50px">
+                        <CodeEditor
+                            isLineNumbersVisible
+                            code={ defaultSubjectTemplate }
+                            onChange={ setSubjectTemplate }
+                            isMinimapVisible={ false }
+                            height="50px"
+                        />
+                    </SkeletonIfLoading>
+                </FormGroup>
+                <FormGroup
+                    label="Body"
```

Comment:
We don't need `subject` and `body` - only 1,- it can be called `template` or `content`